google/cloud/spanner_v1/streamed.py

```diff
@@ -190,6 +190,19 @@ def one_or_none(self):
         except StopIteration:
             return answer
 
+    def fetch_as_dictionary_list(self):
```

Comment:
Name can be better . Probably "to_dictionary" ?

Reply:
to_dictionary would mean it's a dict but the return type here is a list of dict.

Reply:
Simply `dict_list`? Similar to method name `one`?

Reply:
Does to_dict_list look good?